docker-compose.yaml

```diff
@@ -15,6 +15,9 @@ services:
     image: mongo:3.6
     ports:
       - "127.0.0.1:27017:27017"
+    volumes:
+      - "mongo:/data/db"
```

Comment:
Hm, isn't this syntax used to share Volumes between services?
Can't this just be `- "./mongo:/data/db"` or something, without the lines below?

Reply:
Using a relative path is more error prone, since it is more likely for the files to be manipulated.

But we could use an absolute path instead of a named volume.

Reply:
That’s actually what I was trying to aim at :)
Can you do that instead please?